src/index.html

```diff
@@ -12,7 +12,69 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <aside class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
+      <div class="calendar__day"></div>
+
```

Comment:
I think that all the empty lines are redundant)
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
I made empty lines because AI Buddy recommended